packages/shared/src/lib/core/transactions/stores/transactions.store.ts

```diff
@@ -147,6 +148,40 @@ export function addBlockscoutTransactionToPersistedTransactions(
     })
 }
 
+export function addNovesTransactionToPersistedTransactions(
+    profileId: string,
+    accountIndex: number,
+    networkId: NetworkId,
+    newTransactions: NovesTxsResponse
```

Comment:
Is this the correct type, we don't need to store things like pagination? We should just have a list of noves transactions as the type.